index.js

```diff
@@ -85,6 +85,22 @@ Client.prototype.newMonitor = function (options, callback) {
   return this.request('newMonitor', params).nodeify(callback);
 };
 
+Client.prototype.editMonitor = function (options, callback) {
+  if (!options.monitorID) throw new Error('monitorID is required');
+  var params = {};
+    params.monitorID = options.monitorID;
+    if (options.FriendlyName) params.monitorFriendlyName = options.friendlyName;
+    if (options.url) params.monitorURL = options.url;
+    if (options.subType) params.monitorSubType = options.subType;
+    if (options.keywordType) params.monitorKeywordType = options.keywordType;
+    if (options.keywordValue) params.monitorKeywordValue = options.keywordValue;
+    if (options.httpUsername) params.monitorHTTPUsername = options.httpUsername;
+    if (options.httpPassword) params.monitorHTTPPassword = options.httpPassword;
+    if (options.alertContacts) params.alertcontacts = options.alertContacts.join('-');
+    if (options.interval) params.monitorInterval = options.interval;
```

Comment:
The indentation looks off here.  Why not just copy things across in the same style as the `newMonitor` method? `undefined` will be handled equivalently to not setting the property anyway.